cmd/promtool/main.go

```diff
@@ -133,6 +133,11 @@ func main() {
 	dumpPath := tsdbDumpCmd.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
 	dumpMinTime := tsdbDumpCmd.Flag("min-time", "Minimum timestamp to dump.").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
+	importCmd := app.Command("import", "[Experimental] import samples from input and produce TSDB block. Please refer to the storage docs for more details.")
+	importDbPath := importCmd.Flag("output", "output directory for generated block").Default(".").String()
+	importFilePath := importCmd.Flag("input-file", "disables reading from input and using file to import samples from. If empty input is required").String()
+	importBlockSize := importCmd.Flag("block-size", "The maximum block size. The actual block timestamps will be aligned with Prometheus time ranges.").Default("2h").Hidden().Duration()
+	omImportCmd := importCmd.Command("openmetrics", "import samples from OpenMetrics input and produce TSDB block. Please refer to the storage docs for more details.")
```

Comment:
```suggestion
	omImportCmd := importCmd.Command("openmetrics", "Import samples from OpenMetrics input and produce TSDB block(s). Please refer to the storage docs for more details.")
```

Reply:
Wonder if we want command or actually detect the format based on first bytes / file extension / some flag? (:

Reply:
A slippery slope. (o:

I'd go for an explicit option for now.

Reply:
Happy with this. Just wanted to explain that there are other choices (: